components/Cards/examples/supplemental/CardTintExampleCell.m

```diff
@@ -44,4 +46,39 @@ - (void)layoutSubviews {
   _sliderView.center = CGPointMake(self.contentView.center.x, self.bounds.size.height - _sliderView.bounds.size.height);
 }
 
+- (void)prepareForReuse {
+  self.selected = false;
+  _switchView.on = false;
+  _sliderView.value = 0;
+
+  [self resetInteractableDemoValues];
+}
+
+#pragma mark - Accessibility
+
+- (void)customAccessibilityActionToggleSwitch {
+  _switchView.on = !_switchView.isOn;
+}
+
+- (void)accessibilityIncrement {
+  _sliderView.value += _sliderView.maximumValue * 0.25f;
+}
+
+- (void)accessibilityDecrement {
+  _sliderView.value -= _sliderView.maximumValue * 0.25f;
+}
+
+// accessibilityLabel for interactable mode
+- (NSString *)accessibilityLabel {
+  return [NSString stringWithFormat:@"%@ cell. Switch is: %@.",
```

Comment:
Accessibility label should be a single short label and should not end with a period. Maybe something like, "Selected and switched on" or "Off".  I think `UIAccessibilityTraitSelected` is preferred over a custom label.